src/extension/netcdf/src/main/java/org/geoserver/netcdf/NetCDFUnitsConfigurator.java

```diff
@@ -40,7 +40,7 @@ private void configure() {
             LinkedHashMap<String, String> aliases =
                     getMapResource(NETCDF_UNIT_ALIASES, NetCDFUnitFormat.NETCDF_UNIT_ALIASES);
             if (aliases != null) {
-                NetCDFUnitFormat.setAliases(aliases);
+                NetCDFUnitFormat.getInstance().setAliases(aliases);
```

Comment:
The changes look good but [setAliases is private](https://github.com/geotools/geotools/pull/4690/files#diff-ea578c094576d0dbfc82e0a8f1e124d9c468944092fcac75f71e33c79e687121R154) in the GeoTools PR. 

Did you forget to push some changes over there?